wp-multisearch-widget.css

```diff
@@ -151,68 +157,92 @@
 .wrap-askus .askus-name:after {
 	content: ":";
 }
+.wrap-bento-beta {
```

Comment:
I think we should rename this wrapper class to something more like ``wrap-banner-text`` since it won't be bento in the future - but it's not necessary.